<launch>
	<!--<node pkg="pelican" type="tunning_pelican_node" name="tunning_pelican" output="screen">
		<remap from="odometry" to="/msf_core/odometry" />
		<remap from="command/motor_rpm" to="/fcu/command/direct_motor" />
	</node>-->
	<node name="trajectory_gen_node" pkg="traj_gen" type="trajectory_gen_node" output="screen">
		<remap from="odometry" to="/msf_core/odometry" />
	</node>
	<node name="pos_control_node" pkg="position_control" type="pos_control_node" output="screen">
		<remap from="odometry" to="/msf_core/odometry" />
	</node>
	<node name="quat_att_control_node" pkg="attitude_control" type="quat_att_control_node" output="screen">
		<remap from="odometry" to="/msf_core/odometry" />
		<remap from="command/motor_speed" to="/fcu/command/direct_motor" />
	</node>
</launch>
